src/serialize/column.rs

```diff
@@ -911,10 +941,41 @@ mod tests {
         ) -> Result<S::Ok, S::Error> {
             try_serialize::<Position, _>(archetype, &mut out)?;
             try_serialize::<Velocity, _>(archetype, &mut out)?;
+            try_serialize::<i32, _>(archetype, &mut out)?;
             out.end()
         }
     }
 
+    #[test]
+    fn deterministic() {
+        let buffer = Vec::<u8>::new();
+
+        let mut world = World::new();
+        let mut context = Context {
+            components: vec![ComponentId::Position, ComponentId::Velocity]
+        };
+        let mut serializer = serde_json::Serializer::pretty(buffer);
```

Comment:
This is a large increase in dev-dependencies. Why can't we use `serde_test` here like every other test?